src/components/Settings.vue

```diff
@@ -4,35 +4,35 @@
     <v-card-text>
       <OpenImage @change="onImageOpen"
         @delete="ImageDelete"
-        :ImageCheck = "ImageCheck"/>
-      <ImageProcessor :imageUrl = "imageUrl" @imageSampled="onImageSampled"/>
+        :imageCheck="imageCheck"/>
+      <ImageProcessor
+        :image-width="imageWidth"
+        :image-height="imageHeight"
+        :image-url="imageUrl"
+        @imageSampled="onImageSampled"/>
       <v-layout row wrap>
           <v-flex xs3 style="margin: 24px;">
               <v-text-field
               label="Width"
               id="widthSetting"
-              placeholder="32"
+              placeholder="64"
               type="number"
-              value=32
-              @change="onDimChange"
+              v-model="imageWidth"
```

Comment:
```suggestion
              v-model.number.lazy="imageWidth"
```

This will automatically convert the value to a number, and it will not update until the user clicks out (as opposed to typing). Then you don't have to parse the integer in the validation functions.

Reply:
Oh, nice. That's what I was looking for.